lib/train/plugins/transport.rb

```diff
@@ -1,7 +1,4 @@
 # encoding: utf-8
-#
-# Author:: Dominik Richter (<dominik.richter@gmail.com>)
```

Comment:
I am not sure why this is removed?

Reply:
It was a directive via Adam to remove any author tags on files we touch. I can add it back for now.